src/main/java/org/capstone/job_fair/services/impl/job_fair/JobFairServiceImpl.java

```diff
@@ -329,10 +336,30 @@ public void updateJobFairDraft(JobFairDTO dto) {
 
     }
 
+    @Override
+    public CompanyJobFairStatusDTO getJobFairForCompanyById(String companyId, String jobfairId) {
```

Comment:
rename to getJobFairForCompanyByIdAndJobFairID
Why not return an Optional here?